model/api/src/main/java/org/opennms/oce/model/api/ModelObject.java

```diff
@@ -36,12 +36,36 @@
 
     Set<ModelObject> getChildren();
 
+    Set<ModelObject> getPeers();
+
+    Set<ModelObject> getUncles(); // Dependents
+
+    Set<ModelObject> getNephews(); // Dependencies
```

Comment:
Aunts/Uncles:
getParent().getSiblings()?

Nieces/Nephews:
Can we reference these from the siblings of the parents?

The naming of these, Uncles and Nephews just made me giggle a little bit. :)

Reply:
He He.... eeek. I didn't think anyone was going to look at these yet..... ;)
This is the naming convention in Martin's model.
Looking to refine terminology asap but as a group we decided to go forward with the existing for now.
Also no regard for thread safety at this point - just trying to chisel out the rough form without tripping over each other. :)